src/clr-core/button/button.element.spec.ts

```diff
@@ -3,28 +3,114 @@
  * This software is released under MIT license.
  * The full license information can be found in LICENSE in the root directory of this project.
  */
-import { CwcButton } from '@clr/core/button';
+import { ClrLoadingState, CwcButton } from '@clr/core/button';
 import '@clr/core/button';
 import { componentIsStable, createTestElement, removeTestElement, waitForComponent } from '@clr/core/test/utils';
-
 describe('button element', () => {
   let testElement: HTMLElement;
   let component: CwcButton;
+  const placeholderText: string = 'Button Placeholder';
 
   beforeEach(async () => {
     testElement = createTestElement();
-    testElement.innerHTML = `<cwc-button></cwc-button>`;
+    testElement.innerHTML = `<cwc-button>${placeholderText}</cwc-button>`;
 
     await waitForComponent('cwc-button');
     component = testElement.querySelector<CwcButton>('cwc-button');
+
+    // will throw the following error without the uninstall() call:
+    // "Jasmine Clock was unable to install over custom global timer functions. Is the clock already installed?"
+    // See: https://github.com/gruntjs/grunt-contrib-jasmine/issues/213
+    jasmine.clock().uninstall();
```

Comment:
Does `jasmine.clock()` need to be installed before every test? Or can we install for just those that need it?

Reply:
I think it would only be needed in the tests that have async tasks running like `setTimeout` or `setInterval`

Reply:
Agreed. I'll move it into the specific test that needs it.